src/renderer/src/components/add-station-model/station-modal.tsx

```diff
@@ -48,7 +48,7 @@ const AddStationModal: React.FC<AddStationModalProps> = ({ closeModal }) => {
 
             <div className="modal-footer mt-3">
               <button type="button" className="btn btn-danger " onClick={closeHander}>
-                Close
+                Cancel
```

Comment:
Does hitting cancel undo any of the frequencies you might have added in the dialog? If not, close is probably the better term to use.

Reply:
Bruh I don't even think I changed that, blaming it on Copilot told it to rewrite a phrase and its gone rewrote everything

Reply:
Stupid copilot!!!